AnnoyancesFilter/Cookies/sections/cookies_general.txt

```diff
@@ -11,6 +11,7 @@
 !#######################################################
 ! SECTION: Cookies - General element hiding
 !
+##.qc-cmp2-container
```

Comment:
We have a rule for this. Look at the network requests and find similar with this element.

Reply:
You can find the rule in `cookies_specific.txt`.